themes-default/slim/views/partials/showheader.mako

```diff
@@ -163,7 +163,7 @@
                                     <span class="imdbPlot" style="color:#6ae;cursor:pointer">show more..</span>
                                     </div>
                                 % endif
-                                    </td></tr>
+                                    </i></td></tr>
```

Comment:
I hope this is okay, Vue was saying there was a missing ending tag for the `<i>`.